plugins/orchestrator/src/generated/core/ApiError.ts

```diff
@@ -0,0 +1,21 @@
+import type { ApiRequestOptions } from './ApiRequestOptions';
```

Comment:
Do we need to include the automatically generated files in the commits? Other plugins typically exclude them. Since the OpenAPI client generation process is idempotent, I believe it's unnecessary.

Reply:
I'm not a TypeScript expert, so I may be missing something. However, we need to use the generated code in [OrchestratorClient](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator/src/api/OrchestratorClient.ts). If I don't include the generated code in the commits, how can we ensure that it's available for use?

Reply:
+1

Reply:
Apologies, I chose to ask a leading question rather than explicitly explaining my approach. The answer is no, we do not need to include the automatically generated files in the commit. Running `build` from the root directory will invoke `tsc`, which in turn calls `openapi:generate`.

Reply:
@schultzp2020  Ok, now I see your point, thanks for clarifying. 

To better understand: if I don't commit the generated code, when a developer checks out the project they will initially see import errors. However, building the project will generate the necessary code, fixing these errors. Additionally, when creating the `orchestrator` package, the generated code will be included, so everything will work as expected. Correct?

The downside is that developers will see a lot of uncommitted changes. Should I add the generated folder to .gitignore to prevent this? Or is there a better approach?

Reply:
Yes to both questions. https://github.com/janus-idp/backstage-plugins/blob/main/plugins/servicenow-actions/.gitignore

Reply:
I see you used .gitignore :) I'll apply the changes